### PR TITLE
Initial start of comparison table

### PR DESCRIPTION
**Pull request by j1nx**

### CODE REVIEW

**Comment by JarbasAl** [comment]
should also have classic core (picroft) in there

File: docs/comparison.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| Network Setup - Options |
+| Mobile WiFi Setup<br>Easy device "hotspot" to connect to preset network from phone or pad.  | Yes | No | No | Yes |
+| On device WiFi Setup<br>Configure the WiFi connection on the device itself | Yes | Yes | WIP | No |
+| On screen keyboard | Yes | Yes | No | No |
+| Configuration - Option |
+| Data privacy | Yes | Yes | Yes | Partial |
+| Offline mode | Yes | Yes | Yes | No |
+| Color theming | Yes | Yes | Yes | No |
+| Non-Pairing mode | Yes | Yes | Yes | No |
+| API Acces w/o pairing | Yes<br>most skills | Yes<br>most skills | Yes | No |
```
Comment (JarbasAl) [inline docs/comparison.md:22]: isnt it all skills ?

Comment (j1nx) [reply docs/comparison.md:22]: Nope, still working on it. Just pushed for backup purposes and to quickly inspect if i got that MD table markup right.

Comment (j1nx) [reply docs/comparison.md:22]: Oh hang on, realise what you meant. Had that question as well. Just copied it over from the neon table for now.

**Comment by j1nx** [comment]
> should also have classic core (picroft) in there

Thought about that as well. Agreed, but let's call it mycroft-core classic (or sandboxed? Whatever that is)

**Comment by j1nx** [comment]
@NeonDaniel Could you please fill in the "?" and "Unknowns" for Neon.
@AIIX For you the same but for Manjaro.

If any of you would like to add rows or change things over, by all means go ahead.

**Pull request merged**